examples/libf3d/python/img-cmp/img_cmp.py

```diff
@@ -1,19 +1,27 @@
+import sys
 from argparse import ArgumentParser
 
 import f3d
 
-if __name__ == "__main__":
+
+def main(argv=None):
```

Comment:
```suggestion
def main(argv: list[str] | None = None):
```
type hint, same for other `main(argv)` functions